File: Delete Node in a Linked List/DeleteNodeinaLinkedList.cs
namespace LeetcodePracticeCsharpVersion
{
    class DeleteNodeinaLinkedList
    {
        public void DeleteNode(ListNode node)
        {
            if (node == null || node.next == null) return;
            var next = node.next;
            node.next = next.next;
            node.val = next.val;
        }
    }
}

File: 0237_Delete Node in a Linked List/DeleteNodeinaLinkedList.java
/**
 * Definition for singly-linked list.
 * public class ListNode {
 *     int val;
 *     ListNode next;
 *     ListNode(int x) { val = x; }
 * }
 */
class Solution {
    public void deleteNode(ListNode node) {
        if(node == null || node.next == null) return;
        ListNode next = node.next;
        node.next = next.next;
        node.val = next.val;
    }
}
